bakalaurinis.tex

```diff
@@ -0,0 +1,100 @@
+\documentclass{VUMIFInfBakalaurinis}
+\usepackage{algorithmicx}
+\usepackage{algorithm}
+\usepackage{algpseudocode}
+\usepackage{amsfonts}
+\usepackage{amsmath}
+\usepackage{bm}
+\usepackage{caption}
+\usepackage{color}
+\usepackage{float}
+\usepackage{graphicx}
+\usepackage{hyperref}  % Nuorodų aktyvavimas
+\usepackage{listings}
+\usepackage{subfig}
+\usepackage{url}
+\usepackage{wrapfig}
+
+
+% Titulinio aprašas
+\university{Vilniaus universitetas}
+\faculty{Matematikos ir informatikos fakultetas}
+% \institute{Informatikos institutas}  % Užkomentavus šią eilutę - institutas neįtraukiamas į titulinį
+\department{Programų sistemų katedra}
+\title{Daugiaplatformių mobiliųjų aplikacijų kūrimo karkasų analizė}
+\titleineng{Analysis of cross-platform mobile app development frameworks}
+\papertype{Bakalaurinio darbo planas}
+\status{4 kurso 4 grupės studentas}
+\author{Edas Lakavičius}
+% \secondauthor{Vardonis Pavardonis}   % Pridėti antrą autorių
+\supervisor{lekt. Jonas Ragaišis}
+\date{Vilnius \\ \the\year}
+
+% Nustatymai
+% \setmainfont{Palemonas}   % Pakeisti teksto šriftą į Palemonas (turi būti įdiegtas sistemoje)
+\bibliography{bibliografija}
+
+\begin{document}
+\maketitle
+
+\tableofcontents
+
+\section{Tyrimo objektas ir aktualumas}
+Tyrimo objektas - daugiaplatformių mobiliųjų aplikacijų kūrimo karkasai.
+
+Šiais laikais internetas, socialinė medija ir elektroninė prekyba yra neatsiejami nuo kasdienybės dalykai. Interneto naudojimas išaugo nuo 413 klientų milijonų 2000 metais iki 3.4 milijardų 2016 \cite{owidinternet} ir vis dar auga. Didžioji dalis interneto naudotojų yra per mobilius telefonus prisijungiantys klientai, kurie greit lenks kompiuterius naudojančius klientus, net 4 kartus \cite{internetusage}. Būtent dėl to nenuostabu, kad atsiranda vis daugiau projektų, kurie fokusuojasi į mobiliųjų programėlių kūrimą.
+
+Šiuo metu didžiausios aplikacijų parduotuvės "Google Play" ir "Apple App Store" turi daugiau nei 5 milijonų mobiliųjų programėlių \cite{appcount} ir įmonei, norinčiai efektyviai dalyvauti šioje rinkoje reiks ne tik daug investuoti, bet ir gebėti greitai kurti ir gerinti savo aplikacijas. Įprastai yra kuriamos atskiros aplikacijos "iOS" ir "Android" platformom, kas dažniausiai reikalauja turėti atskiras komandas ar programuotojus specializuojančius prie vienos iš šių sistemų, tačiau pačios aplikacijos turi būti kuriamos vienodos.
+
+Būtent dėl didelių kaštų, aplikacijų kūrimo greičio ir skirtingų platformų apjungimo išpopuliarėjo hibridinio tipo technologijos - daugiaplatformių mobiliųjų programėlių kūrimo karkasai. Per trumpą laiką buvo sukurta gana daug skirtingų technologijų skirtų tam pačiam prieš tai jau minėtam tikslui - apjungti skirtingas platformas. Todėl yra aktualu šiuos karkasus išanalizuoti ir palyginti, kad skirtingi projektai galėtų pasirinkti sau tinkamiausią. Šiame darbe lyginsiu tris šiuo metu populiariausius karkasus - "React Native", "Ionic" ir "Flutter" \cite{popularframeworks}.
```

Comment:
Čia rašai, kad aktualu karakasus išanalizuoti ir palyginti. Bet komisija pasakys, kad jau yra daug tinklaraščių, kurie lygina. 
Pvz. Google suvedus pirmas rezultatas: https://betterprogramming.pub/react-native-vs-flutter-vs-ionic-46d3350f96ee#:~:text=Overview%3A%20React%20Native%20is%20based,but%20Ionic%20uses%20multiple%20languages.&text=Ionic%3A%20It%20uses%20multiple%20technologies,to%20use%20native%20platform%20controllers.

Tai klausimas: kuo tavo darbas yra unikalus? T.y. kokią problemą sprendžia? 

Bet ar tikrai šiuo darbu galėsi atsakyti į klausimą, kad skirtingi projektai galėtų pasirinkti kurį nors vieną iš karaksų. Kas jei kuriu žaidimą? O kas jei tai navigacinė programėlė? E-parduotuvė? Socialinis tinklas? Bankinė programėlė? Video pokalbių programėlė? ir t.t. 

Pagal išsikeltą tikslą palyginti tas platformas tiesiog padarysi apžvalgą ir tiek (sužiūrėsi pliusus, minusus, skirtumus, panašumus), tačiau ar tai spręs kokią nors problemą? Nežinau ir manau, kad komisija dėl to irgi turės pastabų.